tests/rip7560/conftest.py

```diff
@@ -22,7 +22,7 @@ def paymaster_contract_7560(w3):
     contract = deploy_contract(
         w3,
         "rip7560/RIP7560Paymaster",
-        value=10 ** 18,
+        value=10**18,
```

Comment:
nitpic: we need some standardized env setting... we keep moving these back and forth.
(or simpler - extract ONE_ETH constant)

Reply:
We have one, We're using Black formatter. I got it as a git hook locally as well. Please use it as well